Assets/Project/Scripts/StageSelectScene/StageSelectDirector.cs

```diff
@@ -113,6 +113,20 @@ private void MakeButtons()
                     // クリック時のリスナー
                     button.GetComponent<Button>().onClick.AddListener(() => StageButtonDown(button));
                     // TODO: ステージを選択できるか、ステージをクリアしたかどうかでButtonの表示を変更する
+
+                    // TODO: stageId get by level data
+                    var stageId = j + 1;
+                    var stageData = GameDataBase.GetStage(stageId);
+                    if (stageData == null)
+                        continue;
+
+                    // ステージ制限なし
+                    if (stageData.stageConstraint == 0)
```

Comment:
`stageConstraint` を `int?` 型（Null 許容型）で宣言して，制約がない時は `null` を入れることはできますか？Editor 拡張を使う場合にできるかどうか知識を持ち合わせていないので，出来るならやって欲しいです．その際に，このコード箇所は，以下のように変えてもらえたらと思います
```suggestion
                    if (stageData.stageConstraint == null)
```

Reply:
`int?` は一度試してみたが、 `EditorGUILayout.PropertyField` でうまく取れないようですので `int` にしました。

Reply:
まあ、どうしても `int?` を使いたいなら方法もあるけど

Reply:
> `int?` は一度試してみたが、 `EditorGUILayout.PropertyField` でうまく取れない

そんな気はした．